loom/test/test_documented.py

```diff
@@ -1,36 +0,0 @@
-# Copyright (c) 2014, Salesforce.com, Inc.  All rights reserved.
```

Comment:
Why was this file removed?

Reply:
@fsaad, I could not get it  to run on Travis. Travis + Conda are moving files around in way that I don't understand and I had no reasonable way to debug what was going on. I guess we could mark it xfail?